cmd/gh-classroom/shared/shared_test.go

```diff
@@ -94,88 +94,3 @@ func TestListAcceptedAssignments(t *testing.T) {
 	assert.Equal(t, "student2", actual.AcceptedAssignments[0].Students[0].Login)
 
 }
-
-func TestListAllAcceptedAssignments(t *testing.T) {
```

Comment:
Is there a reason why this test was deleted? Ideally, we should still have test coverage for this function.

Reply:
@zrdaley Yes I agree having the maximum test coverage is defiantly ideal. However,  I  read the gock documentation to see it it was thread safe and it is not. The [gock documentation](https://github.com/h2non/gock#race-conditions) states "gock is not fully thread-safe, but sensible parts are. Any help making gock **more reliable** in this sense is appreciated." My concern was having tests that may fail intermittently because the mocking framework is not fully thread safe.

If you think gock is reliable enough to use in this case then I can add it back in and just carefully document this issue in the test itself  or if you have another endpoint that we could use to get test data. Let me know your thoughts :)